fred2/sexp_tree.cpp

```diff
@@ -3499,7 +3499,15 @@ int sexp_tree::query_default_argument_available(int op, int i)
 			return 0;
 
 		default:
-			Int3();
+			if (!Dynamic_enums.empty()) {
+				if ((type - First_available_opf_id) < (int)Dynamic_enums.size()) {
+					return 1;
+				} else {
+					UNREACHABLE("Unhandled SEXP argument type!");
+				}
+			} else {
+				UNREACHABLE("Unhandled SEXP argument type!");
+			}
```

Comment:
The qtfred file has a `query_default_argument_available` function (it has two of them, in fact) so this code should be copied to the default case, around line 1690.

Reply:
and done